docs/components/xr-mode-ui.md

```diff
@@ -19,25 +19,52 @@ to the [`<a-scene>` element][scene]. If we wish to simply toggle the UI, use CSS
 
 ## Properties
 
-| Property      | Description                                                         | Default Value |
-|-----------------------|---------------------------------------------------------------------|---------------|
-| cardboardModeEnabled  | Enables the now deprecated cardboard mode.                          | false         |
-| enabled               | Whether or not to display UI related to entering VR.                | true          |
-| enterVRButton         | Selector to a custom VR button. On click, the button will enter VR. | ''            |
-| enterVREnabled        | If the VR button is displayed when applicable                       | false         |
-| enterARButton         | Selector to a custom AR button. On click, the button will enter AR. | ''            |
-| enterAREnabled        | If the AR button is displayed when applicable                       | false         |
-| XRMode                | If the AR, VR button or both will be displayed.  One of 'ar', 'vr' or 'xr'.| vr            |
+| Property                | Description                                                         | Default Value | One of |
+|-------------------------|---------------------------------------------------------------------|---------------|--------|
+| `cardboardModeEnabled`  | Enables the now deprecated cardboard mode.                          | `false`       | -
+| `enabled`               | Whether or not to display UI related to entering VR.                | `true`        | -
+| `enterVRButton`         | Selector to a custom VR button. On click, the button will enter VR. | `''`          | -
+| `enterVREnabled`        | If the VR button is displayed when applicable.                      | `true`        | -
+| `enterARButton`         | Selector to a custom AR button. On click, the button will enter AR. | `''`          | -
+| `enterAREnabled`        | If the AR button is displayed when applicable.                      | `true`        | -
+| `XRMode`                | If the AR button, VR button, or both buttons will be displayed.     | `vr`          | `ar`, `vr`, `xr`
 
 ### Specifying a Custom Enter VR Button
 
+Display only a custom Enter VR button when VR is supported. A custom AR button will not be displayed.
+
+```html
+<a-scene
+  xr-mode-ui="enterVRButton: #myEnterVRButton;">
+  <!-- Style the button with images and/or custom CSS. -->
+  <a id="myEnterVRButton" href="#"></a>
+</a-scene>
+```
+
+### Specifying a Custom Enter AR Button
+
+Display only a custom Enter AR button when AR is supported. A custom VR button will not be displayed.
+
+```html
+<a-scene
+  xr-mode-ui="enterARButton: #myEnterARButton; XRMode: ar;">
+  <!-- Style the button with images and/or custom CSS. -->
+  <a id="myEnterARButton" href="#"></a>
+</a-scene>
+```
+
+### Specifying Custom Enter VR and Enter AR Buttons
+
+Display a custom Enter VR Button when VR is supported and a custom Enter AR button when AR is supported.
+
```

Comment:
@dmarcos I've condensed the three sections back down to one. Hopefully, this is more concise and helps to clarify.